runtime/binding-kafka/src/main/java/io/aklivity/zilla/runtime/binding/kafka/internal/stream/KafkaCacheClientFetchFactory.java

```diff
@@ -887,16 +888,16 @@ private void doClientFanoutReplyWindow(
         private final long replyId;
         private final long leaderId;
         private final long authorization;
-        private final KafkaCacheCursor cursor;
         private final KafkaIsolation isolation;
         private final KafkaDeltaType deltaType;
         private final KafkaOffsetType maximumOffset;
         private final LongSupplier isolatedOffset;
         private final LongSupplier initialGroupIsolatedOffset;
 
+        private KafkaCacheCursor currentCursor;
+        private KafkaCacheCursor newCursor;
```

Comment:
Please revert the `currentCursor` rename back to `cursor`, and change `newCusor` to `nextCursor`.
Please also move `cursor` field back to where it was in the source.

Reply:
I moved the cursor because it's not final anymore. Let me know if you think we should move it back anyway.

Reply:
Good point, this positioning makes total sense then. 👍